static/app/views/integrationPipeline/pipelineView.tsx

```diff
@@ -9,7 +9,7 @@ import {
   QueryClient,
   QueryClientProvider,
 } from 'sentry/utils/queryClient';
-import {GithubInstallationSelect} from 'sentry/views/integrationPipeline/githubInstallationSelect';
+import GithubInstallationSelect from 'sentry/views/integrationPipeline/githubInstallationSelect';
```

Comment:
nit: I personally prefer named imports since they're more IDE-friendly, but that's personal preference and we don't have linting rules for this sort of thing